pyOCD/gdbserver/gdbserver.py

```diff
@@ -332,21 +466,20 @@ def resume(self):
         self.timeOfLastPacket = time()
```

Comment:
I changed resume() to use time_of_last_packet from the packet I/O object, and removed both cases of setting timeOfLastPacket in GDBServer.